### PR TITLE
Aster + Bold. + General fixes

### PR DESCRIPTION
**Pull request by SlyViolin**
- Added Inner Aster Town
- Added Boldarihm (Partial)
- LHG Cleanup ( deleted 76, aligned 11935)
- 4545 Assigned as commodityshop feature
- Partial Wormhole cleanup.

## Checklist

The list below serves as a reminder to make sure, all needed changes were added to the repository. You can add a `x` between the square brackets to tick the items when in edit mode or just tick the checkboxes when not.

If you forgot something, don't worry. Just do the changes on the same branches and they are added automatically to the pull request.

- [x] Added changed map file
- [x] Used a descriptive pull request title
- [x] Wrote a short description of the change in the pull request description (see below). Include area names where possible.
- [x] Added a short sentence summarizing the changes to the `changelog.txt` file
- [x] Increased the number in `version.txt` by 1

### CODE REVIEW

**Comment by ryanstadther** [comment]
Change area "Inside Aster Town" to just "Aster Town" to match IG area name
There are a handful of spots where the layout could be more compact...but not a big deal.
Aster Town is kind of tricky to do on one level. I think it looks pretty good, but we'll see if there is any other feedback!

Other than that...looks great IMO. Thanks!

For reference after this is closed, it will fix #147 #184 #183 and #186!! Good stuff @SlyViolin!

File: Map/changelog.txt [hunk 1]
```diff
@@ -446,4 +446,5 @@ changelog = {
 	[[Fixed continent data for digsites and forays to enable fast travel again]],
 	[[Hesperia Estate fixed, also changed the layout of the Houses area a bit so they don't get all cluttered together and confusing]],
 	[[Added Lupine Hunting Grounds]],
+	[[Added Inner Aster Town, Boldarihm, and General fixes]]
```
Comment (vadi2) [inline Map/changelog.txt]:
```suggestion
	[[Added Inner Aster Town, Boldarihm, and General fixes]],

```

**Comment by ryanstadther** [comment]
Looks like the version number was incremented by 2, but I'm not sure if that matters...as long as it is different I think it should be fine? I'm not entirely sure... @vadi2?

File: Map/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-449
+451
```
Comment (vadi2) [inline Map/version.txt:1]:
```suggestion
450
```

Comment (vadi2) [reply Map/version.txt:1]: Agree @ThaisenLV, it doesn't matter but why start skipping them :)

**Comment by SlyViolin** [comment]
- Inner Aster Town name changed to Aster Town
- Aster Town compacted where I could. Aerek's requested the hill to the keep to remain spread out a little for readability when moving fast.
- Version reverted to 450, though throws an error.
- Aster Town and Boldarihm assigned to respective continents.

**Comment by vadi2** [comment]
What error are you getting with the version?

**Comment by ryanstadther** [comment]
@vadi2  looks like dangerjs checks are throwing an error here on the PR. Not really sure why though...maybe @keneanung has an idea :)

**Comment by ryanstadther** [comment]
All looks good to me now though, thanks @SlyViolin!

**Comment by SlyViolin** [comment]
> What error are you getting with the version?

I throws an error for setting the version to 450, telling me I need to increment by 1. That's why it was set to 451 in the original commit to bypass the danger check. Not sure why it wants me to jump by 2.

**Comment by vadi2** [comment]
Not sure either, nothing looks off on a quick scan. @keneanung would know, though we can just increment by 2 if we don't have the time to look into this particular issue

**Comment by SlyViolin** [comment]
Changed back to 451 just so this can get merged. We can see if the problem happens again next update.

**Review by vadi2 — APPROVED** [review]
Agreed

**Review by ryanstadther — APPROVED** [review]
Looks great!

**Comment by SlyViolin** [comment]
File definitely has wormholes locked but still says there are unlocked wormholes.
https://snipboard.io/stBw4D.jpg

**Comment by keneanung** [comment]
Closing in favour of #194

**Pull request closed without merging**